src/vinyldns/client.py

```diff
@@ -216,9 +223,9 @@ def create_group(self, group, **kwargs):
         :return: the content of the response, which should be a group json
         """
         url = urljoin(self.index_url, u'/groups')
-        response, data = self.__make_request(url, u'POST', self.headers, json.dumps(group), **kwargs)
+        response, data = self.__make_request(url, u'POST', self.headers, to_json_string(group), **kwargs)
 
-        return data
+        return Group.from_dict(data)
```

Comment:
we no longer return dictionaries, we return actual types (horray!).  `xxx.from_dict` should be used for all returns now